crates/nu-command/Cargo.toml

```diff
@@ -100,6 +100,7 @@ uuid = { version = "1.3.0", features = ["v4"] }
 wax = { version = "0.5.0" }
 which = { version = "4.4.0", optional = true }
 print-positions = "0.6.1"
+nix = { version = "0.26.2", default-features = false, features = ["signal"] }
```

Comment:
This should only be part of the feature-gated dependencies below so we don't need to compile that for windows.

Furthermore I wonder if we the `nix` crate provides additional value over the `libc` crate for this usage, if the call is `unsafe`. Is this just a thin wrapper around the syscalls or is there some additional logic to shim platform differences.